# goal definition
string plane_config
---
# result definition
bool success
mcr_perception_msgs/PlaneList recognized_planes
---
# feedback
string current_state
string message 
